### ISSUE TITLE
Fix 'test_feeds/msu/test_invalid_values_msu_feed' tests in 4.2

### ISSUE THREAD

**Issue by damarisg** [wazuh/wazuh-qa#1551]
### Issue information

| Related issue |
|--|
| Closes https://github.com/wazuh/wazuh-qa/issues/1531|

When it comes to starting with the test fix, there is some information that can help you:
```
Module: Vulnerability Detector
Support Team:  "binary beasts"
Target: Manager   
OS: Linux
```
I add some information obtained while researched the logs that failed.

In this case, there 2 different errors when it failed. For that, I will create 2 cases but the test is the same. 
The idea is that you can analyze and fix both.

## Case 1:

|  Type | Description  |
|--|--|
|Test Path | test_feeds/msu/test_invalid_values_msu_feed
| Consistent | yes
| Test Execution | 3/3 executions failed
| Cases Fails | 93
| Summary| On `modify_feed` method when called `read_json_file`. The problem seems to be a Parser error when reading the file.
|Error message| "ERROR at setup of test_invalid_values_msu_feed"

## Case 2:
|  Type | Description  |
|--|--|
|Test Path | test_feeds/msu/test_invalid_values_msu_feed
| Consistent | yes
| Test Execution | 3/3 executions failed
| Cases Fails | 93
| Summary| On `test_invalid_values_msu_feed` when called `check_feed_imported_successfully` the problem could be a Parser error when check in the log or a problem when file is imported.
| Error message |  "Could not find the message: 'Microsoft Security Update' feed finished successfully"

In order to finish this issue the following tasks should be fulfilled:
- [ ] Research of fails.
- [ ] Apply Fix 
- [ ] Full Green/ Full Yellow  in `test_feeds/msu/test_invalid_values_msu_feed` for 3 times.
- [ ] Documentation of any important change done for these tests or the used tools.

**Comment by mdengra** [comment]
### 2021-07-15

Test results with the modules `sca`, `syscollector` and `rootcheck` modules disabled in the `ossec.conf`: 

### Case 1
| Test Executions | Date  | By  | Status | 
|--|--|--|--|
|[test_invalid_values_msu_feed_local_dm_r1.log](https://github.com/wazuh/wazuh-qa/files/6824069/test_invalid_values_msu_feed_local_dm_r1.log)| 2021-07-15 | Miguel | :green_circle: 
|[test_invalid_values_msu_feed_local_dm_r2.log](https://github.com/wazuh/wazuh-qa/files/6824072/test_invalid_values_msu_feed_local_dm_r2.log)| 2021-07-15 | Miguel | :green_circle: 
|[test_invalid_values_msu_feed_local_dm_r3.log](https://github.com/wazuh/wazuh-qa/files/6824078/test_invalid_values_msu_feed_local_dm_r3.log)| 2021-07-15 | Miguel | :green_circle: 

### Case 2
| Test Executions | Date  | By  | Status | 
|--|--|--|--|
|[test_invalid_values_msu_feed_local_dm_r1.log](https://github.com/wazuh/wazuh-qa/files/6824087/test_invalid_values_msu_feed_local_dm_r1.log)| 2021-07-15 | Miguel | :green_circle: 
|[test_invalid_values_msu_feed_local_dm_r2.log](https://github.com/wazuh/wazuh-qa/files/6824090/test_invalid_values_msu_feed_local_dm_r2.log)| 2021-07-15 | Miguel | :green_circle: 
|[test_invalid_values_msu_feed_local_dm_r3.log](https://github.com/wazuh/wazuh-qa/files/6824094/test_invalid_values_msu_feed_local_dm_r3.log)| 2021-07-15 | Miguel | :green_circle:

**Comment by damarisg** [comment]
This issue is closed by #1605

**Comment by damarisg** [comment]
This issue is closed by #1605 

## Test Results from `1531-full-yellow-vuln-det`

| Test Executions | Date  | By  | Status | 
|--|--|--|--|
|[YellowResultsInvalidValuesMSUFeed.log](https://github.com/wazuh/wazuh-qa/files/6856805/YellowResultsInvalidValuesMSUFeed.log)| 2021-07-21| Seyla| 🟡 
|[YellowResultsInvalidValuesMSUFeed2.log](https://github.com/wazuh/wazuh-qa/files/6856806/YellowResultsInvalidValuesMSUFeed2.log)| 2021-07-21| Seyla| 🟢 
|[YellowResultsInvalidValuesMSUFeed3.log](https://github.com/wazuh/wazuh-qa/files/6856807/YellowResultsInvalidValuesMSUFeed3.log)| 2021-07-21| Seyla| 🟡 


| Reference | Status | 
|--|--|
| 🟢 | Pass without Warnings|
| 🟡 | Pass with Warnings |
